### PR TITLE
[incubator-kie-issues-1277] Remove unused bpmn in jbpm-test

### PR DESCRIPTION
**Pull request by elguardian**
Closes : https://github.com/apache/incubator-kie-issues/issues/1277

### CODE REVIEW

**Review by pefernan — APPROVED** [review]
+1 when green

**Comment by kie-ci3** [comment]
**PR job** `#3` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-runtimes -u https://github.com/apache/incubator-kie-kogito-runtimes/pull/3536 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/apache_migration/job/pullrequest_jobs/job/kogito-runtimes-pr/job/PR-3536/3/display/redirect

**Test results:**
- PASSED: 3259
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/apache_migration/job/pullrequest_jobs/job/kogito-runtimes-pr/job/PR-3536/3/testReport/org.kie.kogito.persistence.kafka/KafkaProcessInstancesIT/testFindByIdReadMode/">org.kie.kogito.persistence.kafka.KafkaProcessInstancesIT.testFindByIdReadMode</a></summary>
Container startup failed for image vectorized/redpanda:v21.11.8
</details>

File: jbpm/jbpm-tests/src/test/resources/correlationkey/Process1.bpmn2
Comment (martinweiler) [inline jbpm/jbpm-tests/src/test/resources/correlationkey/Process1.bpmn2:1]: Would having [this test](https://github.com/apache/incubator-kie-kogito-runtimes/commit/b52f369328f543518f884e67d87c52ebec9ca151#diff-8614ba55a71e626287696e63f6d61aab335fb158f47ca350db1b3d8206acf455R2118) add any value now?

Comment (elguardian) [reply jbpm/jbpm-tests/src/test/resources/correlationkey/Process1.bpmn2:1]: unfortunately that is for testing the business key generation. which a feature we don't have a replacement here. Not very sure how to translate it.

Comment (martinweiler) [reply jbpm/jbpm-tests/src/test/resources/correlationkey/Process1.bpmn2:1]: thanks for the clarification

**Review by martinweiler — APPROVED** [review]

**Pull request merged**